ert/serialization/_serializer.py

```diff
@@ -70,3 +71,26 @@ async def decode_from_path(
         async with aiofiles.open(path, mode="rt", encoding="utf-8") as filehandle:
             contents = await filehandle.read()
             return self.decode(contents, *args, **kwargs)
+
+
+class _ecl_sum_serializer(Serializer):
+    def encode(self, obj: Any, *args: Any, **kwargs: Any) -> str:
+        raise NotImplementedError("not implemented")
+
+    def decode(self, series: str, *args: Any, **kwargs: Any) -> Any:
+        raise NotImplementedError("not implemented")
+
+    async def encode_to_path(
```

Comment:
Wondering about the encoding part? Shall this be implemented in transmitter directly (eg. by using tar transformation) or making a more complex yaml file to encode the eclsum file(s)?

Reply:
I am not sure we ever want to implement encoding of EclSum records into new  UNSMRY files. For preservation and usage of summary data outside of ERT, we should provide the binary source files, using e.g. tar yes.